xbmc/games/controllers/guicontrols/GUIGameControllerList.h

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (C) 2022-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "games/GameTypes.h"
+#include "guilib/GUIControl.h"
+#include "guilib/GUIListContainer.h"
+#include "peripherals/PeripheralTypes.h"
+
+#include <memory>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+class CScroller;
+class TiXmlElement;
```

Comment:
Superfluous? im not seeing a need to forward declare this.

Reply:
Good catch